Bottom: 3c98e6ff297326e1cfc0e489cd651ed0c96259fb
Top:    1f7bea9a55e17a2547032e2851a8b113dcd2ec1d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-18 22:08:56 +0000

Refresh of action-c-adjust-some-whitespac

---

diff --git a/src/action.c b/src/action.c
index 6c3ab45..9480355 100644
--- a/src/action.c
+++ b/src/action.c
@@ -2679,7 +2679,7 @@ ActionDisplay (int argc, char **argv, Coord childX, Coord childY)
 
 	case F_CycleClip:
 	  notify_crosshair_change (false);
-	  if TEST_FLAG (ALLDIRECTIONFLAG, PCB)
+	  if (TEST_FLAG (ALLDIRECTIONFLAG, PCB))
 	    {
 	      TOGGLE_FLAG (ALLDIRECTIONFLAG, PCB);
 	      PCB->Clipping = 0;
